subprojects/testing-native/src/integTest/groovy/org/gradle/nativeplatform/test/xctest/SwiftXCTestIntegrationTest.groovy

```diff
@@ -131,7 +130,7 @@ apply plugin: 'swift-library'
         then:
         result.assertTasksExecuted(":compileDebugSwift", ":compileTestSwift", ":linkTest", ":bundleSwiftTest", ":xcTest", ":test")
         assertMainSymbolIsAbsent(objectFiles(lib.test, "build/obj/test"))
-        assertMainSymbolIsAbsent(machOBundle("build/bundle/main/debug/Greeter"))
+        assertMainSymbolIsAbsent(machOBundle("build/exe/test/GreeterTest"))
```

Comment:
Why did this change?

Reply:
I seriously don't know why it was changed or when it was changed. It was just wrong. While I was changing this I remembered that I liked better the old ways where all binaries were under `build/binaries` because the build bundle isn't an `exe`, but isn't an actual bundle. It's the bundled binary which is compiled differently that lib or exe.

Reply:
We need macOS CI!